modules/bullet/rigid_body_bullet.cpp

```diff
@@ -832,7 +832,8 @@ void RigidBodyBullet::on_exit_area(AreaBullet *p_area) {
 
 void RigidBodyBullet::reload_space_override_modificator() {
 
-	if (!is_active())
+	// Make sure that kinematic bodies have their total gravity calculated, even though they aren't active
```

Comment:
Can you please remove this: _"even though they aren't active"_ since the kinematic is not part of simulation and can't be active / unactive

Reply:
Done (via 5fb8a6a)